common/changes/pcln-icons/AIR-53696-Added-checked-carryon-personalItem-icons_2023-05-12-14-45.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "pcln-icons",
+      "comment": "Added new checkedBag, carryOnBag and personalItem icons",
+      "type": "patch"
```

```suggestion
      "type": "minor"
```

additive changes should be minor